testsuite/features/step_definitions/common_steps.rb

```diff
@@ -834,57 +832,21 @@
 end
 
 # Register client
-Given(/^I update the profile of this client$/) do
-  step %(I update the profile of "sle_client")
-end
 
 Given(/^I update the profile of "([^"]*)"$/) do |client|
   node = get_target(client)
   node.run('rhn-profile-sync', timeout: 500)
 end
 
-When(/^I register using "([^"]*)" key$/) do |key|
-  step %(I register "sle_client" as traditional client with activation key "#{key}")
-end
-
-When(/^I register "([^"]*)" as traditional client$/) do |client|
-  step %(I register "#{client}" as traditional client with activation key "1-SUSE-KEY-x86_64")
-end
-
-And(/^I register "([^*]*)" as traditional client with activation key "([^*]*)"$/) do |client, key|
-  node = get_target(client)
-  if client.include? 'sle'
-    node.run('zypper --non-interactive install wget', timeout: 500)
-  else # As Ubuntu has no support, must be CentOS/SLES_ES
-    node.run('yum install wget', timeout: 600)
-  end
-  registration_url = if $proxy.nil?
-                       "https://#{$server.full_hostname}/XMLRPC"
-                     else
-                       "https://#{$proxy.full_hostname}/XMLRPC"
-                     end
-  command1 = "wget --no-check-certificate -O /usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT http://#{$server.full_hostname}/pub/RHN-ORG-TRUSTED-SSL-CERT"
-  # Replace unicode chars \xHH with ? in the output (otherwise, they might break Cucumber formatters).
-  log node.run(command1, timeout: 500).to_s.gsub(/(\\x\h+){1,}/, '?')
-  command2 = "rhnreg_ks --force --serverUrl=#{registration_url} --sslCACert=/usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT --activationkey=#{key}"
-  log node.run(command2, timeout: 500).to_s.gsub(/(\\x\h+){1,}/, '?')
-end
-
 When(/^I wait until onboarding is completed for "([^"]*)"((?: salt minion)?)$/) do |host, is_salt|
```

Comment:
```suggestion
When(/^I wait until onboarding is completed for "([^"]*)"((?: salt minion)?)$/) do |host|
```
`is_salt` is not used anymore.

Reply:
Yeah, that was already fixed when I saw it in Rubocop.